db/migrate/20190627040131_remove_superfluous_translation_override_records.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class RemoveSuperfluousTranslationOverrideRecords < ActiveRecord::Migration[5.2]
+  def up
+    I18n.overrides_disabled do
```

Comment:
I am very uneasy with this migration cause it loads up the entire localization system, in general I prefer migration to simple only run sql statements. 

Can we live with this superfluous data for now? End users I assume will have a simple way to correct if needed.